src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -584,7 +584,8 @@
      {:db (-> db
               (assoc-in [:wallet :ui :send :just-completed-transaction?] true)
               (assoc-in [:wallet :ui :send :transaction-ids] transaction-ids)
-              (assoc-in [:wallet :transactions] (merge wallet-transactions transactions)))
+              (assoc-in [:wallet :transactions] (merge wallet-transactions transactions))
+              (assoc-in [:wallet :force-balance-refresh?] true))
```

Comment:
Are there moments in application state when we call `:wallet/get-wallet-token-for-account` but do not want to refetch balances and okay with getting cached ones?  
If yes then we should cleanup this key when `:wallet/get-wallet-token-for-account` done. 
If no - we do not need this key at all and can pass `true` to all `wallet_fetchOrGetCachedWalletBalances` calls.

Reply:
The thing is we currently dispatch `:wallet/get-wallet-token-for-account` for each account. So if we cleanup  `[:wallet :force-balance-refresh?]` after `:wallet/get-wallet-token-for-account` is done, the rest of the calls will not be forced to refresh.

That said, given the issues reported by @VolodLytvynenko, I think we should always force refresh for now, remove this flag and revisit / discuss the approach to cache balances after this release. cc @vkjr @shivekkhurana

Reply:
@briansztamfater, if we decide to go this way then we can move the `force-refresh?` option inside the `:wallet/get-wallet-token-for-account` and not pass it at all, that will ensure that there are no calls to this event that do not trigger refresh. And later when we revisit it we can change it back.
Wdyt?